internal/flavors/benchmark/aws_org.go

```diff
@@ -103,7 +103,7 @@ func (a *AWSOrg) initialize(ctx context.Context, log *logp.Logger, cfg *config.C
 			return m, nil
 		}))
 
-	return reg, cloud.NewDataProvider(), nil, nil
+	return reg, cloud.NewDataProvider(cloud.WithAccount(*awsIdentity)), nil, nil
```

Comment:
this is needed here -

https://github.com/elastic/cloudbeat/blob/b7a4f0e82cef4abb4e6273a789ac5726b6a12d56/internal/dataprovider/providers/cloud/data_provider.go#L66